filebeat/prospector/prospector.go

```diff
@@ -84,7 +84,7 @@ func New(
 // Start starts the prospector
 func (p *Prospector) Start() {
 	p.wg.Add(1)
-	logp.Info("Starting prospector of type: %v; ID: %d ", p.config.Type, p.ID)
+	logp.Debug("prospector", "Starting prospector of type: %v; ID: %d ", p.config.Type, p.ID)
```

Comment:
I think the trailing space in `ID: %d "` could be dropped (not sure what it was there originally).

Reply:
Done.

Reply:
I think I introduced this because of the `reloading` feature to have some info there on what happens. Is this showing up also in other parts?